src/commands/database/mysql/user/create.tsx

```diff
@@ -0,0 +1,146 @@
+import { ExecRenderBaseCommand } from "../../../../lib/basecommands/ExecRenderBaseCommand.js";
+import {
+  makeProcessRenderer,
+  processFlags,
+} from "../../../../rendering/process/process_flags.js";
+import { ReactNode } from "react";
+import { Flags, Args } from "@oclif/core";
+import { assertStatus } from "@mittwald/api-client-commons";
+import { Success } from "../../../../rendering/react/components/Success.js";
+import { Value } from "../../../../rendering/react/components/Value.js";
+
+type Result = {
+  mysqlUserId: string;
+};
+
+export class Create extends ExecRenderBaseCommand<typeof Create, Result> {
+  static description = "Create a new mysql user";
+  static args = {
+    "database-id": Args.string({
+      description: "ID of the MySQL Database to create a user for",
+    }),
+  };
+  static flags = {
+    ...processFlags,
+    "database-id": Flags.string({
+      required: true,
+      description: "ID of the MySQL Database to create a user for",
+    }),
```

Comment:
For a single command, there is no need to have BOTH an argument and a flag for the same value. Pick one (to stay consistent with other commands like `database mysql create`, which accepts a `--project-id` _flag_, this should probably be a flag).

Reply:
sticking with the flag